docs/apm/apm-alerts.asciidoc

```diff
@@ -56,6 +56,19 @@ Based on the criteria above, define the following rule details:
 * **Has anomaly with severity** - `critical`
 * **Check every** - `5 minutes`
 
+[NOTE]
```

Comment:
I just realised that this description is for `Anomaly rule`. We don't have KQL filter for `Anomaly rule`, so we don't need to add new text for this rule.